src/vuln_analysis/tools/transitive_code_search.py

```diff
@@ -82,17 +82,13 @@ async def _arun(query: str) -> tuple:
 
     yield FunctionInfo.from_fn(
         _arun,
-        description=("This tool is useful when you need to search the container image's code for "
-                     "a given function or method in a given package or library, if it's being used in"
-                     " application or called from application code base. "
-                     "This requires both package name and function/method name, separated by "
-                     "',' as input. Input example - 'package_name,function_name'."
-                     " It returns a tuple of first boolean element indicating whether the function"
-                     " is being called or reachable from an application code base or not, and "
-                     "the second element is a list of documents comprising the call of "
-                     "hierarchy path, showing the calls hierarchy from application "
-                     "to a function in the package,in case the first element value in the tuple is True."
-                     "Do not use this list of documents in subsequent or later checks"))
+        description=(
+            "Checks if a function from a package is reachable from application code through the call chain. "
+            "Input format: 'package_name,function_name' (comma-separated). "
+            "Example: 'urllib,parse'. "
+            "Returns: (is_reachable: bool, call_hierarchy_path: list)."
+        )
+    )
```

Comment:
@etsien I Asked Shimon to extend the description of tool to support more cases, as it's not always the case that it's just a function inside a package ( for example, sometimes in python there is a method of class ( prefixed or not by containing module, depends on the import statement)  that should be checked if it's being called from source code or not, and then the format could be sometimes  <package> , (<module>).<class>.<method>. he should do it as an addition to his transitive code search support work for c programming language.